### PR TITLE
Add spec for date class

### PR DESCRIPTION
**Pull request by m-nakamura145**
Add some specs for `Date` class

* `Date#wday`
* `Date#sunday?`
* `Date#monday?`
* `Date#tuesday?`
* `Date#wednesday?`
* `Date#thursday?`
* `Date#friday?`
* `Date#sturday?`

### CODE REVIEW

File: library/date/wday_spec.rb [hunk 1]
```diff
@@ -2,5 +2,8 @@
 require 'date'
 
 describe "Date#wday" do
-  it "needs to be reviewed for spec completeness"
+  it "returns saturday" do
```
Comment (eregon) [inline library/date/wday_spec.rb]: `returns the week day as a number starting with Sunday as 0`

File: library/date/friday_spec.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require 'date'
+
+describe "Date#friday?" do
+  it "should be friday" do
+    Date.new(2000, 1, 7).friday?.should be_true
+  end
```
Comment (eregon) [inline library/date/friday_spec.rb]: To test this better, it would be interesting to check that `friday?` returns false for dates which are not on friday.

**Comment by eregon** [comment]
@m-nakamura145 Would you have time to take a second look at this?
I would like the specs to be improved a bit before merging.

**Comment by m-nakamura145** [comment]
@eregon I',m sorry to be late. I fixed some specs!

**Comment by eregon** [comment]
No worry, thank you!

**Pull request merged**